python/shotgun_model/shotgun_model.py

```diff
@@ -440,6 +440,7 @@ def _load_data(
                                           each column in the row contains the value for the corresponding field from columns. This means
                                           that the data from the loaded entity will be available field by field. Subclasses can modify
                                           this behavior by overriding _get_additional_columns.
+        :param editable_columns:          A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
The harness is great but there is also value in having some usage examples showing up in the sphinx docs. A lot of people will look [here](http://developer.shotgunsoftware.com/tk-framework-shotgunutils/shotgun_model.html#shotgun_model.ShotgunModel._load_data) and having some cut-past type examples I think would be super useful. Maybe we can mention the test harness? or add a comment to the ticket covering the test harness to make sure we update these docs to include a link to the test harness once it goes public!